de-lib/src/main/java/org/iplantc/de/client/services/DiskResourceServiceFacade.java

```diff
@@ -129,7 +131,7 @@ void moveContents(final Folder sourceFolder,
      * @param dest id of the destination folder.
      * @param callback service success/failure callback
      */
-    void importFromUrl(String url, DiskResource dest, AsyncCallback<String> callback);
+    void importFromUrl(String url, DiskResource dest, DECallback<String> callback);
```

Comment:
This one is + may turn into even more of a weird case; URL import right now goes from terrain to apps and doesn't use data-info. But, the plan is to have it go through terrain into data-info, which then calls out to apps (CORE-6214). I'm not sure how things go down in these cases.